resources/aws_ec2.go

```diff
@@ -52,6 +52,8 @@ const (
 	SnsPrefix = Ec2Prefix + "sns-"
 	// SnsTopicName is the name of the sns topic created by snsMakeTopic.
 	SnsTopicName = SnsPrefix + "events"
+	// httpProto is used to tell sns that the subscriber uses the http protocol.
+	SnsSubscriptionProto = "http"
```

Comment:
is there an https version?
also comment should start with `SnsSubscriptionProto` instead.

Reply:
There is, but it requres a valid ssl certificate, which I don't have. In practice we should be able to detect which protocol the user wants to use by the WatchEndpoint URL they specify. I will edit the first commit to change the comment.